reporter/src/main/resources/templates/asciidoc/default.ftl

```diff
@@ -106,23 +101,20 @@ resolvedLicenses =
 ]
 [#if resolvedLicenses?has_content]
 
-The following copyrights and licenses were found in the source code of this package:
+The following licenses and copyrights were found in the source code of this package:
```

Comment:
Out of scope: I wonder if the found licenses do include curations and if so if it's fine to still say "found in the source code".